braindecode/datautil/serialization.py

```diff
@@ -0,0 +1,85 @@
+"""
+Convenience functions for storing and loading of windows datasets.
+"""
+
+# Authors: Lukas Gemein <l.gemein@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os
+
+import mne
+import pandas as pd
+
+from ..datasets.base import BaseConcatDataset, WindowsDataset
+
+
+def store_windows_dataset(path, windows_dataset, overwrite):
+    """Store braindecode WindowsDatasets to files
+
+    Parameters
+    ----------
+    path: str
+        directory to which .fif and .json files are stored
+    dataset: BaseConcatDataset of WindowsDatasets
+    overwrite: bool
+        whether to overwrite existing files
+    """
+    for ds_i, ds in enumerate(windows_dataset.datasets):
+        ds.windows.save(os.path.join(path, "{}-epo.fif".format(ds_i)),
+                        overwrite=overwrite)
+    windows_dataset.description.to_json(os.path.join(path, "description.json"))
+
+
+def recover_windows_dataset(path, ids_to_load=None):
```

Comment:
the more common naming are

load_ / read_
save_ / write_

cf numpy, mne etc.